images/windowsserver2022/windows-server-2022.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Per HashiCorp, HCL2 is the preferred way to write Packer configurations. Please convert this over